src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private String code;
+
+    Operation(String code) {
+
+        this.code = code;
```

Comment:
The constructor for the enum does not need an empty line at the start. Please remove the empty line at line 12 according to the checklist.